toil-appliance/test/test.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
I would have a preference for running with `set -x -v` since this is a test script. It will make it easier to debug any test failures, should they occur.